zellij-server/src/thread_bus.rs

```diff
@@ -32,14 +30,15 @@ impl ThreadSenders {
                 .unwrap_or_else(|| Ok(()));
             Ok(())
         } else {
-            self.to_screen.as_ref().unwrap().send(instruction)
+            self.to_screen
+                .as_ref()
+                .unwrap()
+                .send(instruction)
+                .context("failed to send message to screen")
```

Comment:
Do you think we can get rid of the `unwrap()` attached to `as_ref`, too? Now that we already return a `Result` we may as well report that error case.

Also what do you think about adding the instruction we sent to the error message, similar to how you already do in `send_to_plugin`? It may not be helpful for the user, but maybe a developer can make sense of all the content.

Reply:
upd: crossbeam::channel::SendError doesn't format the content so yes, that pattern would be helpful for devs. Also I've noticed you've added it as a function on the type so I've moved the usage of `.to_anyhow()` into those `thread_bus` functions, hope you don't mind